tests/test_GCF.py

```diff
@@ -0,0 +1,44 @@
+"""
+Оформите задачу 4.7 в виде функции и покройте ее тестами.
+Учтите, что в функцию могут быть переданы некорректные значения,
+здесь может пригодится ‘assertRaises’.
+Не нужно переделывать функцию для того чтобы она ловила
+все возможные ситуации сама.
+"""
+
+
+import ddt
+from src.GCF import gcf
+import unittest
+
+
+@ddt.ddt
+class GCFTests(unittest.TestCase):
+    def test_gcf(self):
+        self.assertEqual(gcf(99, 11), 11)
+
+    @ddt.data(
+        (30, 18, 6),
+        (39, 16, 1),
+        (44, 16, 4),
+        (99, 18, 9),
+        (9, 9, 9),
+        (0, 0, 0),
```

Comment:
your code doesn't work correctly. we can't have expected divisor - 0. ZeroDivisionError